_job_offers/bending-spoons.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: event
+title:  "Bending Spoons — Many positions"
+date:   2019-10-15T12:00:00+02:00
+categories: hiring
+slug: 'bending-spoons'
+---
+
+<a target="_blank" href="//bendingspoons.com/">Bending Spoons</a> is the 1st IOS app developer in Europe. Despite being such a young company, we've already achieved notable results, with hundreds of millions of downloads of our apps, and millions of users active every month. Far from being satisfied, we look with excitement at the future, and believe that through smart and hard work we’ll accomplish much more.
```

Comment:
why `<a></a>` instead of `[...](https://...)`?

Reply:
Because we want to open it in a new tab